smac_planner/test/deprecated_upsampler/upsampler_cost_function_nlls.hpp

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) 2020, Samsung Research America
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License. Reserved.
+
+#ifndef DEPRECATED_UPSAMPLER__UPSAMPLER_COST_FUNCTION_NLLS_HPP_
```

Comment:
why would we be including something "deprecated" in a first merge?

Reply:
Because I spent a stupid amount of time on it and I'd like to preserve it in case we need it in the future. This may be resurrected in the future.